client-app/modules/purchase-requests/components/from-file.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <VcWidget :title="!isMobile ? $t('purchase_request.from_file.title') : undefined" size="md">
```

Comment:
We use BEM for new components, "from-file" BEM block

Reply:
Postponed as require a lot of additional time